src/scripts/main.js

```diff
@@ -2,7 +2,20 @@
 
 const people = require('./lib/people');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const row = document.querySelector('tr');
 
-// write your code here
+for (const person of people) {
+  const sex = person.sex === 'm' ? 'Male' : 'Female';
```

Comment:
```suggestion
```
No need to create a variable for this. You can write it in your `insertAdjacentHTML` method